rust-toolchain.xxx

```diff
@@ -0,0 +1,5 @@
+[toolchain]
```

Comment:
This file should be named `rust-toolchain.toml` so that it can be read by `cargo`. Otherwise, syntax is correct and I like that you specified the required targets. 👍🏻

Reply:
Done :+1: